pkgs/tools/admin/eksctl/default.nix

```diff
@@ -0,0 +1,40 @@
+{ stdenv, buildGoPackage, fetchFromGitHub }:
+
+buildGoPackage rec {
+  pname = "eksctl";
+  version = "0.1.35";
+
+  src = fetchFromGitHub {
+    owner = "weaveworks";
+    repo = "eksctl";
+    rev = version;
+    sha256 = "0b3s7vh85k68wawmsdp96q9l4yhikwhyjn1c7cwxys0aia4i8wkv";
+  };
+
+  goPackagePath = "github.com/weaveworks/eksctl";
+
+  buildPhase =
```

Comment:
Do you need the custom buildPhase?
I left it out and it builds fine - I have no AWS however, so I can't test.